projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -443,7 +450,7 @@ public function update_publicize_connection( $request ) {
 			$body['shared'] = $shared;
 		}
 
-		$response = Client::wpcom_json_api_request_as_user(
+		Client::wpcom_json_api_request_as_user(
```

Comment:
Does it matter if this is unsuccessful? I think we're changing the response so it always returns that it's successful, but tha actual connection may not have been modified. This means that the caller would need to check the connection data that has been returned and can't rely on the status code.

Reply:
I've added error handling here.